src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java

```diff
@@ -43,14 +47,16 @@ public void onMessageDelete(MessageDeleteEvent event) {
     }
 
     private Predicate<Message> isOurResponseTo(long messageId) {
-        String tracker = "re: " + messageId;
+        String tracker = longToBinaryBlankString(messageId);
         return message -> !message.getEmbeds().isEmpty()
                 && message.getAuthor().isYourself()
                 && message.getEmbeds().get(0).getFooter()
                 .flatMap(EmbedFooter::getText)
-                .map(tracker::equals)
+                .flatMap(text -> Optional.of(text.startsWith(tracker)))
```

Comment:
that line can be replaced by `.filter(text -> text.startsWith(tracker))`, no need to create a new `Optional` for that

Reply:
I wanted to do that, but IntelliJ told me following: 
"no instance(s) of type variable(s) U exist so that Boolean conforms to Optional<? extends U>"

Reply:
sounds like you did something wrong or intellij messed up for some reasons. `filter` has no generic parameter `U`:
`Optional<T> filter(Predicate<? super T> predicate)`

Reply:
Yeah, I did it with flatMap and not with filter :)
Thanks.